packages/storybook/src/generators/configuration/util-functions.ts

```diff
@@ -533,3 +536,27 @@ export function rootFileIsTs(
     return tsConfiguration;
   }
 }
+
+export function getE2EProjectName(
+  tree: Tree,
+  mainProject: string
+): string | undefined {
+  let e2eProject: string;
+  forEachExecutorOptions(
+    tree,
+    '@nrwl/cypress:cypress',
+    (options, projectName) => {
+      if (e2eProject) {
+        return;
+      }
+      const { project, target } = parseTargetString(options['devServerTarget']);
```

Comment:
You should check that `options.devServerTarget` is there. If not, it will be `undefined` and this will error.

Reply:
Ohhh shoot!